javascript/iot-sub.js

```diff
@@ -0,0 +1,6 @@
+mqttClient.subscribe('device:rocket:1');
+mqttClient.on('message', (topic, message) => {
+  if (topic === 'action') {
+    rocket.processInstruction(JSON.parse(message));
```

Comment:
Same here

Reply:
Not sure what mqtt client means here?

Reply:
mqttClient is an MQTT Client. See https://github.com/mqttjs/MQTT.js/. There is no need for us to tell developers, in a demo, which MQTT client is being used.  But that is compatible with a real MQTT client's API.